docs/chapters/sdk/installing-the-sdk.rst

```diff
@@ -0,0 +1,29 @@
+Downloading and installing the SDK
+==================================
+
+Download the SDK for your hardware from the `PELUX website`_, it is quite big because it contains tho whole sysroot for the project in addition to some native tools, so depending on your internet connectivity it could take a couple of minutes.
```

Comment:
* It's not the same sentence after '_,' so it should be a '.' really. Same for ', so'.
* IMHO: "it is quite big because" -> "The SDK is a very huge file as".
* "a couple of minutes" -> "from a couple of minutes to a few hours"?